doc/userguide/partials/options.rst

```diff
@@ -92,11 +92,17 @@
    With the -s option you can set a file with signatures, which will
    be loaded together with the rules set in the yaml.
 
+   It is possible to use globbing when specifying rules files.
+   For example, -s /path/to/rules/*.rules
+
 .. option:: -S <filename.rules>
 
    With the -S option you can set a file with signatures, which will
    be loaded exclusively, regardless of the rules set in the yaml.
 
+   It is possible to use globbing when specifying rules files.
+   For example, -S /path/to/rules/*.rules
```

Comment:
I had to use the following to avoid Suri thinking the resulting shell expansion was a bpf filter
`-S '/path/to/rules/*.rules`

Reply:
Will get this updated and resubmit. Thanks for the feedback!